mlir/test/mlir-miopen-lib/populate_bww.mlir

```diff
@@ -48,6 +48,6 @@
 // TUNING3_0: globalSize=2048, localSize=64
 // TUNING3_1: globalSize{{.*}}localSize{{.*}}
 // TUNING3_2: globalSize=2048, localSize=64
-// DRIVER3: miopen.conv2d_bwd_weight(%arg0, %arg1, %arg2, %arg3) features = xdlops {arch = "gfx908", dilations = [1 : i32, 1 : i32], filter_layout = ["g", "k", "c", "y", "x"], gemm_id = 0 : i32, input_layout = ["ni", "gi", "ci", "hi", "wi"], numCu = 120 : i32, output_layout = ["no", "go", "ko", "ho", "wo"], padding = [0 : i32, 0 : i32, 0 : i32, 0 : i32], strides = [1 : i32, 1 : i32]} : memref<1x1024x1024x1x1xf16>, memref<64x1x1024x14x14xf16>, memref<64x1x1024x14x14xf16>, memref<1x1024x1024x1x1xf32>
-// DRIVER3: miopen.conv2d_bwd_weight(%arg0, %arg1, %arg2, %arg3) features = xdlops {arch = "gfx908", dilations = [1 : i32, 1 : i32], filter_layout = ["g", "k", "c", "y", "x"], gemm_id = 1 : i32, input_layout = ["ni", "gi", "ci", "hi", "wi"], numCu = 120 : i32, output_layout = ["no", "go", "ko", "ho", "wo"], padding = [0 : i32, 0 : i32, 0 : i32, 0 : i32], strides = [1 : i32, 1 : i32]} : memref<1x1024x1024x1x1xf16>, memref<64x1x1024x14x14xf16>, memref<64x1x1024x14x14xf16>, memref<1x1024x1024x1x1xf32>
-// DRIVER3: miopen.conv2d_bwd_weight(%arg0, %arg1, %arg2, %arg3) features = xdlops {arch = "gfx908", dilations = [1 : i32, 1 : i32], filter_layout = ["g", "k", "c", "y", "x"], gemm_id = 2 : i32, input_layout = ["ni", "gi", "ci", "hi", "wi"], numCu = 120 : i32, output_layout = ["no", "go", "ko", "ho", "wo"], padding = [0 : i32, 0 : i32, 0 : i32, 0 : i32], strides = [1 : i32, 1 : i32]} : memref<1x1024x1024x1x1xf16>, memref<64x1x1024x14x14xf16>, memref<64x1x1024x14x14xf16>, memref<1x1024x1024x1x1xf32>
+// DRIVER3: miopen.conv2d_bwd_weight(%arg0, %arg1, %arg2, %arg3) features = xdlops|dot|atomic_add {arch = "gfx908", dilations = [1 : i32, 1 : i32], filter_layout = ["g", "k", "c", "y", "x"], gemm_id = 0 : i32, input_layout = ["ni", "gi", "ci", "hi", "wi"], numCu = 120 : i32, output_layout = ["no", "go", "ko", "ho", "wo"], padding = [0 : i32, 0 : i32, 0 : i32, 0 : i32], strides = [1 : i32, 1 : i32]} : memref<1x1024x1024x1x1xf16>, memref<64x1x1024x14x14xf16>, memref<64x1x1024x14x14xf16>, memref<1x1024x1024x1x1xf32>
```

Comment:
Also, not sure 'xdlops' is a good name, how about 'mfma' like 'dot'

Reply:
Yeah, good point, I can tweak it.